force-app/main/default/pages/editAccountName.page

```diff
@@ -16,7 +16,7 @@
     <h1>Edit Account Name</h1><br/><br/>
     <apex:form id="theForm">
         <apex:inputField id="accountNameField" value="{!Account.Name}" />
-        <br/>
+        <br/> 
         <apex:commandButton id="save" action="{!save}" value="Save" onclick="event.preventDefault(); Dialog.confirm('Are you sure?', {ok: 'OK', cancel: 'Cancel'}, (ret) => { if(ret) validateAndSubmit(this); })" />
```

Comment:
After meeting with you, I'm confused with how this works when we're using an apex:commandButton. Does the action actually get called? For example what if you change the action to {!delete}. Does the record get deleted?

Reply:
I updated the comments to cover the scenario in this question.